### DISCUSSION TITLE
Lane size

### DISCUSSION THREAD

**Discussion by NourEldin-Ali** [csgoh/processpiper#47] [Q&A]
Hello,

Thank you for this interesting library.

I tried to create a BPMN diagram like the following image:
![image](https://github.com/csgoh/processpiper/assets/61513661/5bd45b33-f822-4d86-95cf-372cd061eb18)

However, the result I obtained is:
![image](https://github.com/csgoh/processpiper/assets/61513661/a4611efa-dfbf-491b-be25-7552a3a8d57a)

How can I adjust the lane size to match the layout of the first BPMN diagram? or, what steps should I take to ensure a well-designed BPMN diagram?
Note that I use one lane in my process, and I use `auto_size=True`.

Thank you!

**Comment by csgoh** [comment]
Hi @NourEldin-Ali Would you be able to share your code? I suspect it is a defect in the library, but keen to test it at my end.
